src/nvim/ops.c

```diff
@@ -2927,13 +2926,14 @@ void do_put(int regname, yankreg_T *reg, int dir, long count, int flags)
      */
     if (y_type == MCHAR && y_size == 1) {
       do {
-        totlen = count * yanklen;
+        assert(count <= INT_MAX);
```

Comment:
I ran the tests with a `assert(count >= 0)` at the start of the function and nothing happened. This seems to never happen.